pkgs/development/python-modules/ots-python/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, opentype-sanitizer
+, setuptools-scm
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "ots-python";
+  version = "9.0.0";
+
+  src = fetchPypi {
+    pname = "opentype-sanitizer";
+    inherit version;
+    hash = "sha256-S4lmDhZqH+BXqxxrYwOPxI7nPnaOgOG/mCq697FvoGQ=";
+  };
+
+  patches = [
+    ./0001-use-packaged-ots.patch
```

Comment:
Similar to a different comment, use `substituteAll` nix function, and comment something near the patch.